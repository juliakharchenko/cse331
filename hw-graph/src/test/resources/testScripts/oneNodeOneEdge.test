# Tests that a reflexive edge can be added

# Create a graph
CreateGraph reflexiveGraph

# Add a Node
AddNode reflexiveGraph n1

# Add an Edge
AddEdge reflexiveGraph n1 n1 e1

# Print the nodes in the graph and the outgoing edges from n1
ListNodes reflexiveGraph
ListChildren reflexiveGraph n1