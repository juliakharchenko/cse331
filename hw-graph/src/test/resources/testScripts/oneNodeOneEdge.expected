# Tests that a reflexive edge can be added

# Create a graph
created graph reflexiveGraph

# Add a Node
added node n1 to reflexiveGraph

# Add an Edge
added edge e1 from n1 to n1 in reflexiveGraph

# Print the nodes in the graph and the outgoing edges from n1
reflexiveGraph contains: n1
the children of n1 in reflexiveGraph are: n1(e1)